react-client/src/components/GameOutcome.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class GameOutcome extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    identities = []
+    // Here, will have to itterate over players, creating <Identity> compenents.
+
+    return (
+      <div>
+
+        <h5> Game Outcome  </h5>
+
+         Here, we seem to have missed some components of use. I thus inaguate them:
+
+         {identities}
+
+         <NewGameButton />
+         // Unclear from discussion if Buttons should be components
```

Comment:
probably don't need a button component since it would just re-direct players to before the "start game" page